acme.go

```diff
@@ -66,6 +67,13 @@ const (
 	boulderBuildIDPath    = "/build"
 )
 
+var (
+	// ErrUnsupportedRSABitLen reports whether an unrecognized RSA key size was used
+	ErrUnsupportedRSABitLen = errors.New("unsupported RSA bit length")
```

Comment:
Sadly, it turns out Boulder doesn't actually support RSA account keys bigger than 2048 bits or ECDSA account keys bigger than 384 bits. https://github.com/letsencrypt/boulder/blob/master/wfe/jose.go#L12-L27 I created https://github.com/letsencrypt/boulder/issues/1592 to find out why they are not allowed. 

In the mean time, I'm going to comment those specific algorithms / curves.